### ISSUE TITLE
Terrible issues in Keras version=2.2.0 : while_loop() got an unexpected keyword

### ISSUE THREAD

**Issue by yywangvr** [keras-team/keras#10631]
# Hi, When I update keras from 2.1.6 to 2.2.0, I get the problem : 
> while_loop() got an unexpected keyword argument 'maximum_iterations'

# The code is 👍 
'''python
input_img = Input(shape=(120,115))
encoding_dim = 1

encoded = LSTM(encoding_dim)(input_img)

decoded = RepeatVector(120)(encoded)
decoded = LSTM(115, return_sequences=True)(decoded)

autoencoder= Model(inputs=[input_img], outputs=[decoded])
encoder = Model(inputs=[input_img], outputs=[encoded])
autoencoder.compile(optimizer='adam', loss='mse')
encoder.compile(optimizer='adam', loss='mse')
'''
# Please, check what happens :( . Thank you very much.!!!
The problem is also reported [here in this link](https://github.com/tensorflow/tensorflow/issues/20075)!

**Comment by VanitarNordic** [comment]
if you use Anaconda, you can use  `conda install -c hesi_m tensorflow ` to install tensorflow 1.9.0 for CPU 

if you specifically use Keras also, you can use  `conda install -c hesi_m keras` which installs Keras 2.2.0 in accompany with everything including tensorflow 1.9.0

It is better to make a fresh anaconda environment for your experiments

**Comment by lhdgriver** [comment]
I guess you are using tf1.4. Please follow @VanitarNordic 's instructions to update your tf.
To my experience, tf1.5 is ok.
